xl2times/__main__.py

```diff
@@ -485,6 +485,20 @@ def run(args: argparse.Namespace) -> str | None:
 
     model.files.update([Path(path).stem for path in input_files])
 
+    processing_order = ["base", "subres", "trade", "demand", "scen", "syssettings"]
```

Comment:
Should this be defined inside the DataModule class? Just so that if we add another enum value to it, we might be reminded to also add it to this list.

Reply:
Sounds like a good idea! Would you be up for including this in the refactoring of the class in a subsequent PR or do you think we should change it already?